ProjectLighthouse/Types/Serialization/GameChallenge.cs

```diff
@@ -0,0 +1,285 @@
+﻿using System.Collections.Generic;
+using System.Xml.Serialization;
+
+namespace LBPUnion.ProjectLighthouse.Types.Serialization;
+
+[XmlRoot("Challenge_header")]
+public class GameChallenge : ILbpSerializable
+{
+    [XmlElement("Total_challenges")]
+    public int TotalChallenges { get; set; }
+
+    [XmlElement("Challenge_End_Date")]
+    public ulong EndDate { get; set; }
+
+    [XmlElement("Challenge_Top_Rank_Bronze_Range")]
+    public decimal TopRankBronzePercentage { get; set; }
+
+    [XmlElement("Challenge_Top_Rank_Silver_Range")]
+    public decimal TopRankSilverPercentage { get; set; }
+
+    [XmlElement("Challenge_Top_Rank_Gold_Range")]
+    public decimal TopRankGoldPercentage { get; set; }
```

Comment:
```suggestion
    public float GoldRankPercentage { get; set; }
```
These don't need to be decimals if they only represent percentages